package.json

```diff
@@ -16,6 +16,10 @@
     "coverage": "jest --coverage --forceExit --passWithNoTests",
     "prepare": "husky install"
   },
+  "engines": {
```

Comment:
We need to update `README`, `package-lock.json` and `@types/node` as described on separate chat

Reply:
updated read me with new `node`and `npm`versions, 
generate `package-lock.json`,
update `types/node`